compiler/x/runtime/VirtualGuardRuntime.cpp

```diff
@@ -25,8 +25,6 @@
 
 #define IS_32BIT_SIGNED(x)   ((x) == ( int32_t)(x))
 
-extern "C" void _patchingFence16(void *startAddr);
-
```

Comment:
Please update the copyright header in this file.

Reply:
Thank you for pointing it out. Updated.